tests/consent_no_response.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { test, expect, Page } from "@playwright/test";
+
+let p: Page;
+
+test("Consent - No response", async ({ page }) => {
+  p = page;
+  await given_the_app_is_setup();
+
+  await when_i_go_to_the_triage_page();
+  await and_i_click_the_consent_tab();
+  await and_i_click_on_a_patient();
```

Comment:
What do you think about rolling steps like these up into a single step? For example: https://github.com/nhsuk/record-childrens-vaccinations/blob/add-moar-e2e-tests/tests/consent_gillick_refused.spec.ts#L9-L11

```suggestion
  await when_i_select_a_child_with_no_consent_response();
```